resources/css/ceres-item.scss

```diff
@@ -126,86 +116,78 @@ $green:                     #28a745 !default;
 $teal:                      #20c997 !default;
 $cyan:                      #17a2b8 !default;
 
-// legacy
-$brand-primary:             $blue !default;
-$brand-secondary:           $red !default;
-$brand-success:             $green !default;
-$brand-info:                $cyan !default;
-$brand-warning:             $yellow !default;
-$brand-danger:              $red !default;
-
-$primary:                   $brand-primary !default;
-$secondary:                 $brand-secondary !default;
-$success:                   $brand-success !default;
-$info:                      $brand-info !default;
-$warning:                   $brand-warning !default;
-$danger:                    $brand-danger !default;
+$primary:                   $blue !default;
+$secondary:                 $gray-600 !default;
+$success:                   $green !default;
+$info:                      $cyan !default;
+$warning:                   $yellow !default;
+$danger:                    $red !default;
 $light:                     $gray-100 !default;
 $dark:                      $gray-800 !default;
 
 $yiq-text-dark:             $gray-900 !default;
 $yiq-text-light:            $white !default;
-$yiq-contrasted-threshold:  186 !default;
+$yiq-contrasted-threshold:  150 !default;
 
 
 //
 // Sizes
 //
 $heights: (
-  1: (xs: 36px, sm: 48px, md: 64px, lg: 72px, xl: 80px),
-  2: (xs: 72px, sm: 90px, md: 128px, lg: 144px, xl: 160px),
-  3: (xs: 128px, sm: 160px, md: 192px, lg: 224px, xl: 256px),
-  4: (xs: 200px, sm: 244px, md: 288px, lg: 332px, xl: 376px),
-  5: (xs: 272px, sm: 328px, md: 384px, lg: 440px, xl: 496px)
+    1: (xs: 36px, sm: 48px, md: 64px, lg: 72px, xl: 80px),
+    2: (xs: 72px, sm: 90px, md: 128px, lg: 144px, xl: 160px),
+    3: (xs: 128px, sm: 160px, md: 192px, lg: 224px, xl: 256px),
+    4: (xs: 200px, sm: 244px, md: 288px, lg: 332px, xl: 376px),
+    5: (xs: 272px, sm: 328px, md: 384px, lg: 440px, xl: 496px)
 ) !default;
 //
 // Fonts
 //
-$root-directory:            '..' !default;
+$root-directory:            ".." !default;
 
-$flag-icon-css-path:        '#{$root-directory}/images/flags' !default;
-$flag-icon-rect-path:       '/4x3' !default;
-$flag-icon-square-path:     '/1x1' !default;
+$flag-icon-css-path:        "#{$root-directory}/images/flags" !default;
+$flag-icon-rect-path:       "/4x3" !default;
+$flag-icon-square-path:     "/1x1" !default;
 
 $fa-font-path:              "#{$root-directory}/documents/fonts" !default;
 
 //
 // Top Bar
 //
-$top-bar-bg:                $gray-dark;
+$top-bar-bg:                $gray-900;
```

Comment:
die top-bar hat durch diese Änderung nicht den korrekten Farbwert #212529 (war vorher #4D4F4D).
Auf dem Screenshot kann man gut erkennen, dass der to-top button einen helleren Farbton hat als die Topbar:

![Bildschirmfoto 2020-01-13 um 07 59 56](https://user-images.githubusercontent.com/22615263/72237633-78432e00-35db-11ea-89ec-7dbcea6154c9.png)

Reply:
Das haben wir bewusst in Kauf genommen. Ziel war es, die veralteten $gray-(light|lighter|dark|darker)-Variablen durch die Bootstrap-Variablen $gray-100 - $gray-900 zu ersetzen.

Reply:
👍